.gitattributes

```diff
@@ -30,5 +30,4 @@
 /phpstan.neon.dist export-ignore
 /schema.graphql export-ignore
 /node_modules export-ignore
-/packages export-ignore
```

Comment:
Instead of ignoring the `/packages` directory when downloading the plugin from github, this includes it, as it's needed for running `npm install && npm run build`